packages/unified-latex-util-arguments/libs/gobble-single-argument.ts

```diff
@@ -212,3 +220,69 @@ function partialStringMatch(node: Ast.Node, token: Ast.Node): boolean {
         node.content.length > token.content.length
     );
 }
+
+function cloneStringNode(node: Ast.String, content: string): Ast.String {
+    return Object.assign({}, node, { content });
+}
+
+// Find the position of the open brace and the closing brace.
+// Returns undefined if the brace isn't found.
+// This may mutate `nodes`.
+function findBracePositions(nodes: Ast.Node[], startPos: number,
```

Comment:
This function looks to duplicate functionality already in `gobbleSingleArgument`

Reply:
Ah, I see you moved this function outside of the body.